src/containers/Questions/questions.jsx

```diff
@@ -0,0 +1,215 @@
+      }
+      message.warning("You didn't make an entry today");
+      history.push('/home');
+    }
+  };
+
+  render() {
+    const { errors, current, title, content } = this.state;
+    const {
+      history: { goBack },
+    } = this.props;
+    return (
+      <div>
+        <div className="question__navigation">
+          {current > 0 ? (
+            <Icon type="left" onClick={() => this.prev()} />
+          ) : (
+            <BackButton handleBack={goBack} />
+          )}
+          <div>
+            <Popconfirm
+              title="Do you really want to exit?"
+              onConfirm={this.confirm}
+              onCancel={this.cancel}
+              okText="Yes"
+              cancelText="cancel"
+            >
+              <Icon type="close" />
+            </Popconfirm>
+          </div>
+        </div>
+        <div className="question__type">
+          <p>
+            Question 3·/
+            {current + 1}
+          </p>
+          <Progress
+            percent={entryData[current].percent}
+            size="small"
+            showInfo={false}
+          />
+          <p>{entryData[current].heading}</p>
+          <div className="question__title">
+            <Input
+              placeholder="Title"
+              name="title"
+              value={title}
+              autoComplete="off"
+              onChange={this.handleChange}
+            />
+            {errors.title && (
+              <span className="question__error-field">{errors.title}</span>
+            )}
+          </div>
+          <div className="question__content">
+            <TextArea
+              rows={4}
+              placeholder="Write your words"
+              name="content"
+              value={content}
+              onChange={this.handleChange}
+            />
+            {errors.content && (
+              <span className="question__error-field">{errors.content}</span>
+            )}
+          </div>
+        </div>
+
+        <div className="question__steps-action">
+          {current < entryData.length - 1 && (
+            <Button type="primary" onClick={() => this.next()}>
+              Next
+            </Button>
+          )}
+          {current === entryData.length - 1 && (
+            <Button
+              type="primary"
+              onClick={() => {
+                this.finish();
+              }}
+            >
+              Done
+            </Button>
+          )}
+
+          {current < entryData.length - 1 ? (
+            <Button style={{ marginLeft: 8 }} onClick={() => this.skip()}>
+              Skip
+            </Button>
+          ) : (
+            <Button style={{ marginLeft: 8 }} onClick={() => this.skip()}>
+              Skip
+            </Button>
+          )}
+        </div>
+      </div>
+    );
+  }
+}
+
+Questions.propTypes = {
+  history: propTypes.shape({
+    push: propTypes.func.isRequired,
+    goBack: propTypes.func.isRequired,
+  }).isRequired,
+};
```

Comment:
WOW 214 Line 
I think we agreed to put all logic on index.js file 
and here just the jsx code

Reply:
try to do that